tests/PHPUnit/Integration/WidgetsListTest.php

```diff
@@ -37,7 +37,7 @@ public function testGet()
         // check if each category has the right number of widgets
         $numberOfWidgets = array(
             'VisitsSummary_VisitsSummary'  => 6,
-            'Live!'                        => 4,
+            'Live!'                        => 5,
```

Comment:
It is already there, under https://github.com/piwik/piwik/blob/2.16.0-rc1/plugins/Widgetize/tests/System/WidgetTest.php#L178

I've only changed its rendering method to Report instead of using API and injecting data to template directly.